birdhouse/components/geoserver/default.env

```diff
@@ -22,7 +22,10 @@ export GEOSERVER_STABLE_EXTENSIONS="grib-plugin,netcdf-plugin,netcdf-out-plugin,
 
 # Install the community edition plugins specified in
 # https://github.com/kartoza/docker-geoserver/blob/master/build_data/community_plugins.txt
-export GEOSERVER_COMMUNITY_EXTENSIONS="geopkg-plugin"
+export GEOSERVER_COMMUNITY_EXTENSIONS="geopkg-plugin,\
+ogcapi-coverages-plugin,ogcapi-dggs-plugin,ogcapi-features-plugin,ogcapi-images-plugin,\
+ogcapi-maps-plugin,ogcapi-styles-plugin,ogcapi-tiled-features-plugin,ogcapi-tiles-plugin,\
+stac-datastore-plugin"
```

Comment:
Can they be placed line-by-line for easier readability?

Reply:
Sure I can do this for other plugins that we will enable in the future.  For the ogcapi bunch, I consider them as a "group" so I'd rather keep them on the same lines if you don't mind.

Reply:
I think the "group" is more obvious with the "ogcapi-" prefix that would align on all lines when following each other.

Reply:
Knowing you do not like long line, that's why I broke it down to 2 lines of "ogcapi-".  Do you really want me to put it on one line?

Reply:
Or I can keep 2 lines with a comment to group related plugins on the same line, else if they are unrelated, on different line for readability?

Reply:
Since the `ogcapi-` items are already spread out across 2 lines because there are too many entries, I would rather have everything on their own line. 

Also, just noticed that https://github.com/kartoza/docker-geoserver/blob/master/build_data/community_plugins.txt is still referenced above the variable. This is not valid anymore because they switched to `develop` branch with a new plugin download strategy. The old `master` reference only contains `ogcapi-plugin` as a whole, which is misleading. Maybe https://build.geoserver.org/geoserver/ should be used as reference instead.

Reply:
Done in 6282c9c052c5c13932fa400a80e89bcdfde7c6ff.